### DISCUSSION TITLE
Onclick attached function doesn't work on htmlId{}

### DISCUSSION THREAD

**Discussion by milpgame** [KaTeX/KaTeX#3875] [Q&A] [answered]
I have created html zones on math expressions with htmlId{}, I render them on node.js server and on client(Chrome Android Browser) I attach arrow functions to handle the clicks, all works fine, but on zones at \frac denominator , bases of powers ( powers are multiplied at denominator) , the clicks functions doesn't work.

**Comment by edemaine** [comment, marked as answer]
This doesn't sound like a bug in KaTeX, but possibly an issue with what you're trying to do with KaTeX. But we'd need more details (e.g. links to deployed site, or the source code you're using) to help debug.

**Reply by milpgame** [reply]
frac {  { 2 } ^{ 3 }  cdot  {  htmlId{z0}{  fcolorbox{green}{white}{$ 8 $} } } ^{ 5 }  } {  { 2 } ^{ 2 }   cdot  {  htmlId{z1}{  fcolorbox{green}{white}{$ 8 $} } } ^{ 2 }  } 

I can't select 8 from the denominator
Thanks
